tables.sql

```diff
@@ -95,69 +94,79 @@ CREATE TABLE "transactions" (
 );
 
 INSERT INTO feedback
-(customer_id, item_id, comments)
+(customer_id, item_id, comments, rating_review)
 VALUES
-(546, 41, "Snowshoes fit me well."),
-(546, 33, "I like the product. I have nothing to add."),
-(548, 47,  "I like the product. I have nothing to add."),
-(561, 51, "I like the product. I have nothing to add."),
-(550, 52,  "I like the product. I have nothing to add."),
-(555, 35, "I like the product. I have nothing to add."),
-(552, 31, "I like the product. I have nothing to add."),
-(563, 39, "I like the product. I have nothing to add."),
-(570, 36, "I like the product. I have nothing to add."),
-(570, 37, "I like the product. I have nothing to add."),
-(569, 32, "I like the product. I have nothing to add."),
-(564, 49, "I like the product. I have nothing to add."),
-(560, 34, "Road bike is great!"),
-(558, 38, "The  tent is easy to set up!"),
-(565, 40,"The  tent is easy to set up!"),
-(566, 42, "The  tent is easy to set up!"),
-(567, 43, "The bear canister is too small"),
-(568, 32, "I like the product. I have nothing to add."),
-(551, 46, "I like the product . I have nothing to add."),
-(553, 27, "I like the product. I have nothing to add."),
-(554, 48, "I like the product. I have nothing to add."),
-(556, 28, "Snowshoes are too loose for me "),
-(549, 50, "I like the product. I have nothing to add"),
-(557, 29, "I like the product. I have nothing to add."),
-(559, 45,"I like the product. I have nothing to add."),
-(558, 44, "I like the product. I have nothing to add.");
+(1, 15, "Snowshoes fit me well.", 5), 
+(1, 7, "Bike gears are not working properly", 1),
+(3, 21,  "I like the product. I have nothing to add.", 4),
+(16, 25, "I like the product. I have nothing to add.", 5 ),
+(5, 26,  "I like the product. I have nothing to add.", 5),
+(10, 9, "I like the product. I have nothing to add.", 5),
+(7, 5, "I like the product. I have nothing to add.", 5),
+(18, 13, "I like the product. I have nothing to add.", 4),
+(25, 10, "I like the product. I have nothing to add.", 4),
+(25, 11, "I like the product. I have nothing to add.", 4),
+(24, 6, "I like the product. I have nothing to add.", 5),
+(19, 23, "I like the product. I have nothing to add.", 5),
+(15, 8, "Road bike is great!", 4),
+(13, 12, "The  tent is easy to set up!", 5),
+(20, 14,"The  tent is easy to set up!", 5),
+(21, 16, "The  tent is easy to set up!", 4),
+(22, 17, "The bear canister is too small", 2),
+(23, 6, "I like the product. I have nothing to add.", 5),
+(6, 20, "I like the product . I have nothing to add.", 4),
+(8, 1, "I like the product. I have nothing to add.", 4),
+(9, 22, "I like the product. I have nothing to add.", 4),
+(11, 2, "Snowshoes are too loose for me ", 2),
+(4, 24,"I like the product. I have nothing to add", 5),
+(12, 3, "I like the product. I have nothing to add.", 4),
+(14, 19,"I like the product. I have nothing to add.", 5),
+(13, 18, "I like the product. I have nothing to add.", 4);
+
+
+SELECT * 
```

Comment:
This SQL is missing syntax around the UNION.

Reply:
When executing this statement on a newly created Sqlite Db with the script, I get this error.
Statement:
```
SELECT * 
FROM items;
UNION
SELECT AVG(rating_review)
FROM feedback;
```
Error: 
```
Execution finished with errors.
Result: near "UNION": syntax error
UNION
```